tests/behaviour/expectations/behaviour.ts

```diff
@@ -342,6 +342,19 @@ export const expectations = flatten(
             Expect.Simple('doWhile_break', ['0', '2'], ['2']),
           ]),
         ]),
+        new Dir('returns', [
+          File.Simple('returnInserter', [
+            Expect.Simple('test1', ['6'], ['0']),
```

Comment:
These tests could do with more descriptive names so that people can tell which case each one is testing for without having to code and analyse the contract code. It would also be helpful to add a tag to the Expects that use the same function